src/index.html

```diff
@@ -42,45 +42,45 @@ <h3>
         </article>
       <article class="addActivity__article">
-        <form method="POST">
+        <form>
           <label for="form__control--date1">Date
-          <input aria-required="true" id="form__control--date1" type="text" name="date" placeholder="YYYY/DD/MM">
+          <input class="form-control" aria-required="true" id="form__control--date1" type="text" name="date" placeholder="YYYY/DD/MM">
           </label>
           <label for="form__control--steps">Number of Steps
-          <input aria-required="true" id="form__control--steps" type="number" name="number" pattern="/^[0-9]+$/" min="1" max="20000" placeholder="#">
+          <input class="form-control" aria-required="true" id="form__control--steps" type="number" name="number" pattern="/^[0-9]+$/" min="1" max="20000" placeholder="#">
           </label>
           <label for="form__control--minutes">Minutes Active
-          <input aria-required="true" id="form__control--minutes" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="400" placeholder="#">
+          <input class="form-control" aria-required="true" id="form__control--minutes" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="400" placeholder="#">
           </label>
           <label for="form__control--stairs">Flights of Stairs
-          <input aria-required="true" id="form__control--stairs" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="500" placeholder="#">
+          <input class="form-control" aria-required="true" id="form__control--stairs" type="number" name="number" pattern="/^[0-9]+$/" min="1.0" max="500" placeholder="#">
           </label>
-          <button type="submit">Submit</button>
+          <button id="activity-submit" type="submit">Submit</button>
         </form>
       </article>
       <article class="addHydration__article">
-        <form method="POST">
+        <form>
             <label for="form__control--date2">Date
-            <input aria-required="true" id="form__control--date2" type="text" name="date" placeholder="YYYY/DD/MM">
+            <input class="form-control" aria-required="true" id="form__control--date2" type="text" name="date" placeholder="YYYY/DD/MM">
             </label>
             <label for="form__control--water">Ounces of Water Drank
-            <input aria-required="true" id="form__control--water" type="number" name="number" pattern="[-+]?[0-9]*[.,]?[0-9]+" step=0.01 min="1" max="200" placeholder="#">
+            <input class="form-control" aria-required="true" id="form__control--water" type="number" name="number" pattern="[-+]?[0-9]*[.,]?[0-9]+" step=0.01 min="1" max="200" placeholder="#">
             </label>
-            <button type="submit">Submit</button>
+            <button id="hydration-submit" type="submit">Submit</button>
           </form>
       </article>
       <article class="addSleep__article">
-        <form method="POST" >
+        <form>
             <label for="form__control--date3">Date
```

Comment:
Good idea to remove the POST method. I didn't have it set up correctly to work and I don't think it is a good use case for this project.